acceptance/bundle/artifacts/whl_prebuilt_outside/this_dab/databricks.yml

```diff
@@ -0,0 +1,21 @@
+bundle:
+  name: python-wheel
+
+sync:
+  paths:
+  - ../other_dab/dist/*.whl
```

Comment:
This is incorrect syntax, it has to be 

```
sync:
  paths:
   - ..
```

sync -> paths indicated a new sync root, not files to include I believe